xbmc/video/VideoDatabase.cpp

```diff
@@ -6304,7 +6304,21 @@ bool CVideoDatabase::GetSetsByWhere(const std::string& strBaseDir, const Filter
       return false;
 
     CFileItemList sets;
-    GroupAttribute groupingAttributes = ignoreSingleMovieSets ? GroupAttributeIgnoreSingleItems : GroupAttributeNone;
+    GroupAttribute groupingAttributes;
+    const CUrlOptions::UrlOptions& options = videoUrl.GetOptions();
+    auto option = options.find("includesetswithonemovie");
```

Comment:
should this new option `includesetswithonemovie` get documented (doxy/wiki/both) ? @AlwinEsch

Reply:
Would be good to have on both, as it relate to parts outside.

Not sure, think in doxygen the whole show of available variables like "videodb://movies/sets/?includesetswithonemovie=true" not descripted :thinking:.

Reply:
AFAIK none of these options are documented. Back when most of these were introduced we didn't even have doxygen properly setup / integrated.